delivery_recipe/kick_starting_a_project.md

```diff
@@ -0,0 +1,64 @@
+# Kick Starting a Project
+
+Once a project has been sold the next step is usually an intense discovery process to map out processes, to look at possible solutions and to identify the high level requirements (epics). At the end of the discovery process the next step is to hand the client over to the project delivery team to begin the process of developing what’s been agreed.
```

Comment:
I'm not sure they would. I've never been involved in a discovery. Are you saying this is the way you want it to be? For example, you done the JARS discovery then the delivery team picked that up. It'd be good to have a proper handover to the delivery team, to make it clear that when, for example, you have done the discovery, you are not part of the delivery team. With JAC that caused problems with Alan keep referring back to you.